This PR proposes changes that would break the pipeline definition:
```
gardener-extension-provider-vsphere-test-gcve_automation: Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/concourse/replicator.py", line 140, in render
    definition_descriptor = self._render(definition_descriptor)
  File "/usr/lib/python3.10/site-packages/concourse/replicator.py", line 184, in _render
    'definition': factory.create_pipeline_definition(),
  File "/usr/lib/python3.10/site-packages/concourse/factory.py", line 88, in create_pipeline_definition
    self._apply_traits(variant)
  File "/usr/lib/python3.10/site-packages/concourse/factory.py", line 172, in _apply_traits
    for step in transformer.inject_steps():
  File "/usr/lib/python3.10/site-packages/concourse/model/traits/publish.py", line 398, in inject_steps
    for img in self.trait.dockerimages():
  File "/usr/lib/python3.10/site-packages/concourse/model/traits/publish.py", line 294, in dockerimages
    image_dict = self.raw['dockerimages']
KeyError: 'dockerimages'

```

--- a/.ci/pipeline_definitions
+++ b/.ci/pipeline_definitions
@@ -49,6 +49,7 @@ gardener-extension-provider-vsphere:
           branch: 'master'
       steps:
         test-integration:
+          timeout: "4h"
           trait_depends:
             - publish
           image: 'eu.gcr.io/sap-se-gcr-k8s-private/vsphere-private/vsphere-gcve-tm-run:latest'

--- a/.ci/pipeline_definitions
+++ b/.ci/pipeline_definitions
@@ -34,19 +34,6 @@ gardener-extension-provider-vsphere:
                     username: 'briantopping'
                   - type: 'emailAddress'
                     email: 'brian.topping@sap.com'
-          gardener-extension-gcve-tm-run:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/sap-se-gcr-k8s-private/vsphere-private/vsphere-gcve-tm-run'
-            dockerfile: 'Dockerfile'
-            target: gardener-extension-gcve-tm-run
-            tag_as_latest: true
-            resource_labels:
-              - name: 'cloud.gardener.cnudie/responsibles'
-                value:
-                  - type: 'githubUser'
-                    username: 'briantopping'
-                  - type: 'emailAddress'
-                    email: 'brian.topping@sap.com'
   jobs:
     head-update:
       traits:
@@ -102,3 +89,34 @@ gardener-extension-provider-vsphere:
               channel_name: 'C02DYTGSUNQ' #sap-tech-gardener-on-vmware
               slack_cfg_name: 'scp_workspace'
         component_descriptor: ~
+
+gardener-extension-provider-vsphere-test:
+  template: 'default'
+  base_definition:
+    repo: ~
+    traits:
+      version:
+        preprocess: 'inject-commit-hash'
+        inject_effective_version: true
+  jobs:
+    gcve-test-container:
+      traits:
+        component_descriptor: ~
+        draft_release: ~
+        options:
+          public_build_logs: true
+        publish:
+          dockerimages:
+            gardener-extension-gcve-tm-run:
+              registry: 'gcr-readwrite'
+              image: 'eu.gcr.io/sap-se-gcr-k8s-private/vsphere-private/vsphere-gcve-tm-run'
+              dockerfile: 'Dockerfile'
+              target: gardener-extension-gcve-tm-run
+              tag_as_latest: true
+              resource_labels:
+                - name: 'cloud.gardener.cnudie/responsibles'
+                  value:
+                    - type: 'githubUser'
+                      username: 'briantopping'
+                    - type: 'emailAddress'
+                      email: 'brian.topping@sap.com'

--- a/.ci/terraform/main.tf
+++ b/.ci/terraform/main.tf
@@ -214,10 +214,12 @@ provider "nsxt" {
 resource "nsxt_policy_ip_block" "block1" {
   display_name = "ip-block1"
   cidr         = cidrsubnet(local.privatecloud_cred_obj["privateCloud"]["networkconfig"]["managementcidr"], 5, 2)
+  depends_on   = [google_container_node_pool.primary_nodes]
 }
 
 resource "nsxt_policy_ip_pool" "pool1" {
   display_name = "snat-ippool" # this is used by .ci/terraform/charts/testmachinery-secrets/templates/secrets.yaml
+  depends_on   = [google_container_node_pool.primary_nodes]
 }
 
 resource "nsxt_policy_ip_pool_block_subnet" "block_subnet1" {
@@ -226,4 +228,5 @@ resource "nsxt_policy_ip_pool_block_subnet" "block_subnet1" {
   block_path          = nsxt_policy_ip_block.block1.path
   size                = 8
   auto_assign_gateway = false
+  depends_on          = [google_container_node_pool.primary_nodes]
 }

